src/main/java/uk/gov/hmcts/reform/hmc/service/CftHearingServiceImpl.java

```diff
@@ -28,6 +30,8 @@ public class CftHearingServiceImpl implements CftHearingService {
     private final ApplicationParams applicationParams;
     private final SecurityUtils securityUtils;
 
+    public static final String LATEST_HEARING_REQUEST_VERSION = "latestHearingRequestVersion";
```

Comment:
```suggestion
    public static final String LATEST_HEARING_REQUEST_VERSION = "Latest-Hearing-Request-Version";
```
HTTP headers tend to follow a convention more like this. I appreciate this will require updating in the CFT Hearing Service too.

Reply:
Sorted